### ISSUE TITLE
Could not create connection:

### ISSUE THREAD

**Issue by dwx1994** [FACEGOOD/FACEGOOD-Audio2Face#74]
ERROR: Handshake status 429 Too Many Requests
ERROR: Could not create connection: ws://api.tgenie.cn/runtime/v3/recognize?res=comm&productId=914020983&token=f6a315c2-64ed-4e0e-b3a0-b0d69b50144b

**Comment by Jerry-1127** [comment]
I have the same problem as you, did you solve it?

**Comment by dwx1994** [comment]
Yes, I have successfully implemented the logic and ported it to Unity.

**Comment by thd-ux** [comment]
how do you solve it?

**Comment by dwx1994** [comment]
Please go to the following address in the documentation: https://login.tgenie.cn/, and apply for a product account to replace the one in your project.

**Comment by huhuwind** [comment]
Could you tell me where the replacement place is? please please please

**Comment by dwx1994** [comment]
![飞书20230506-172456](https://user-images.githubusercontent.com/58616521/236615675-afdd5bf1-83c2-40ec-acb0-3d5e4cb137a6.jpg)

**Comment by huhuwind** [comment]
> 

Thank you so much. I don't know how to express my thanks.

**Comment by huhuwind** [comment]
Sorry to bother you again, I had such a problem after I changed it like you taught. And the avatars couldn't answer my questions.Perhaps you have ever encountered such a problem?
![image](https://user-images.githubusercontent.com/128301163/236665496-dca59187-78bf-49f0-abfa-b4f53b47b193.png)

**Comment by dwx1994** [comment]
This is what I directly modified for use in Unity
窦卫湘
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "FACEGOOD/FACEGOOD-Audio2Face"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年5月7日(星期天) 下午4:04
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [FACEGOOD/FACEGOOD-Audio2Face] Could not create connection: (Issue #74)





 
Sorry to bother you again, I had such a problem after I changed it like you taught. And the avatars couldn't answer my questions.Perhaps you have ever encountered such a problem?

**Comment by dwx1994** [comment]
![unity](https://user-images.githubusercontent.com/58616521/236719072-7a943f22-d8c7-47d0-9fda-7d6568610e40.png)


This is what I directly modified for use in Unity

**Comment by huhuwind** [comment]
你好，如果您不介意的话，我想要加一下您的联系方式，因为我对于这项工作该如何开展完全没有头绪，希望能够得到一些更加详细的指导。请放心，是有偿的。

**Comment by dwx1994** [comment]
wechat 978763392

**Comment by weicheng-xu** [comment]
Hello, I have the same problem as you, did you solve it?  
![image](https://github.com/FACEGOOD/FACEGOOD-Audio2Face/assets/80668226/fc6ed213-3757-4339-ba3b-90d222ec304e)

**Comment by huhuwind** [comment]
> 

删掉# -*- coding: UTF-8 -*-，保存之后重新运行一下试试

**Comment by weicheng-xu** [comment]
非常感谢，但是还是报这个错

**Comment by LeoRepeater** [comment]
> > 
> 
> 删掉# -- _编码： UTF-8 --_，保存之后重新运行一下试试



> > 
> 
> 删掉# -_- coding: UTF-8 -_-，保存之后重新运行一下试试

请问您解决掉这个问题了吗，具体如何操作。不胜感激

**Comment by zhangsirya** [comment]
> > > 
> > 
> > 
> > 删掉# -- _编码： UTF-8 --_，保存之后重新运行一下试试
> 
> > > 
> > 
> > 
> > 删掉# -_- coding: UTF-8 -_-，保存之后重新运行一下试试
> 
> 请问您解决掉这个问题了吗，具体如何操作。不胜感激

请问您解决这个问题了吗

**Comment by zhangsirya** [comment]
> 非常感谢，但是还是报这个错

请问您解决这个问题了吗

**Comment by huhuwind** [comment]
很抱歉，没有



---Original---
From: ***@***.***&gt;
Date: Wed, Jul 19, 2023 09:11 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [FACEGOOD/FACEGOOD-Audio2Face] Could not create connection:(Issue #74)




  
非常感谢，但是还是报这个错
  
请问您解决这个问题了吗

**Comment by crazy124W** [comment]
在会话精灵里选择智能语音api接入，似乎可以解决上述问题

**Comment by selifecn** [comment]
> 在会话精灵里选择智能语音api接入器，似乎可以解决上述问题

确实可以对话了，不过只能成功对话一次，马上就出现429错误

**Comment by liyuli1997** [comment]
> Please go to the following address in the documentation: https://login.tgenie.cn/, and apply for a product account to replace the one in your project.

Can't register an account.
